test_scripts/API/LastMileNavigation/SubscribeWayPoints/010_SubscribeWayPoints_Transfering_of_HMI_error_resultCode_to_mobile.lua

```diff
@@ -0,0 +1,47 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/26
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/embedded_navi/Subscribe_to_Destination_and_Waypoints.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [SubscribeWayPoints] As a mobile app I want to be able to subscribe on notifications about
+-- any changes to the destination or waypoints.
+--
+-- Description:
+-- In case:
+-- 1) mobile application sent valid and allowed by Policies SubscribeWayPoints_request to SDL
+--
+-- SDL must:
+-- 1) transfer SubscribeWayPoints_request_ to HMI
+-- 2) respond with <resultCode> received from HMI to mobile app
+
+---------------------------------------------------------------------------------------------------
```

Comment:
@LitvinenkoIra Rework test script according to discussed approach with 3 types of result codes:
- successful
- erroneous
- undefined